tests/test_dr.py

```diff
@@ -53,23 +53,20 @@ def test_ingredients(self):
         )
 
     def test_instructions(self):
-        return self.assertEqual(
-            """Opskriften giver cirka 16 styks, afhængig af størrelsen.
-
-I en foodprocessor kommes mel, farin, maizena og salt. Kør til alt er blandet.
-
-Kom smørret ved og kør til det ligner brødkrummer. Tilsæt koldt vand og æggeblomme og kør til dejen lige samler sig.
-
-Kom dejen ud på et melet bord og saml den hurtigt. Kom køkkenfilm omkring den og lad den hvile i køleskabet i en times tid.
-
-Rul den derpå ud på et melet bord og placer den i bunden af en firkantet bageform (24 x 24 cm.) der er foret med bagepapir. Prik dejen med en gaffel og bag den i en 180 grader varm ovn i ca. 20 minutter.
-
-Til karamellen kommes den kondenserede mælk i en gryde sammen med farin, smør, sirup, vanille og salt. Bring massen i kog og lad den koge roligt til den når 110 grader på et sukkertermometer - eller til massen er tyk og gyldenbrun. Hæld karamellem over den stadig lune kagebund.
-
-Lad derpå kagen køle helt af.
+        expected_instructions = "\n".join(
+            [
+                "Opskriften giver cirka 16 styks, afhængig af størrelsen.",
+                "I en foodprocessor kommes mel, farin, maizena og salt. Kør til alt er blandet.",
+                "Kom smørret ved og kør til det ligner brødkrummer. Tilsæt koldt vand og æggeblomme og kør til dejen lige samler sig.",
+                "Kom dejen ud på et melet bord og saml den hurtigt. Kom køkkenfilm omkring den og lad den hvile i køleskabet i en times tid.",
+                "Rul den derpå ud på et melet bord og placer den i bunden af en firkantet bageform (24 x 24 cm.) der er foret med bagepapir. Prik dejen med en gaffel og bag den i en 180 grader varm ovn i ca. 20 minutter.",
+                "Til karamellen kommes den kondenserede mælk i en gryde sammen med farin, smør, sirup, vanille og salt. Bring massen i kog og lad den koge roligt til den når 110 grader på et sukkertermometer - eller til massen er tyk og gyldenbrun. Hæld karamellem over den stadig lune kagebund.",
+                "Lad derpå kagen køle helt af.",
+                "Smelt chokoladen i et vandbad sammen med smørret, bland godt og hæld så chokoladen ovenpå karamellen. Jævn ud med en spatel. Sæt kagen på køl.",
+                "Tag kagen ud af formen og fjern bagepapiret. Skær kagen i små firkanter som opbevares på køl til de skal serveres.",
+            ]
+        )
 
-Smelt chokoladen i et vandbad sammen med smørret, bland godt og hæld så chokoladen ovenpå karamellen. Jævn ud med en spatel. Sæt kagen på køl.
+        actual_instructions = self.harvester_class.instructions()
 
-Tag kagen ud af formen og fjern bagepapiret. Skær kagen i små firkanter som opbevares på køl til de skal serveres.""",
-            self.harvester_class.instructions(),
-        )
+        return self.assertEqual(expected_instructions, actual_instructions)
```

Comment:
```suggestion
    return self.assertEqual(expected_instructions, self.harvester_class.instructions())
```

(I think we can omit the `actual_instructions` variable here)

Reply:
Agreed, use this for debugging I'll swap it back now.